src/stack.js

```diff
@@ -1,5 +1,40 @@
-'use strict'
-
 export default class Stack {
-  // your code here
+
+  constructor() {
+    this._size = 0
+    this.container = []
+  }
+
+  push(data) {
+    this.container[this._size++] = data
+  }
+
+  pop() {
+    if(this._size !== 0){
+      this.container.slice(this._size--)
+    }
+    else{
+      return null
+    }
+  }
+
+  peek() {
+    if (this._size !== 0) {
+      return this.container[this._size - 1]
+    }
+    return null
+  }
+
+  isEmpty() {
+    if (this._size === 0) {
```

Comment:
you could make this a ternary operator:
isEmpty(){return this._size === 0 ? true : false}